posthog/async_migrations/test/test_definition.py

```diff
@@ -23,3 +29,22 @@ def test_get_async_migration_definition(self):
         self.assertEqual(example_migration.operations[-1].fn, example_fn)
         self.assertEqual(example_migration.operations[-1].rollback_fn, example_rollback_fn)
         self.assertTrue(isinstance(example_migration.service_version_requirements[0], ServiceVersionRequirement))
+
+    def test_get_migration_instance_and_parameters(self):
+        set_instance_setting("ASYNC_MIGRATIONS_SHOW_PERSON_ON_EVENTS_MIGRATION", True)
```

Comment:
wrong PR maybe but just came to mind that rather than having to remember to do this on every test we could just have the `is_shown` function account for `settings.TEST`?

Reply:
I could see myself spending some time trying to figure out why my test won't work only to realize this